code/controllers/subsystem/vote.dm

```diff
@@ -74,17 +68,11 @@ SUBSYSTEM_DEF(vote)
 			else if(mode == "map")
 				for (var/non_voter_ckey in non_voters)
 					var/client/C = non_voters[non_voter_ckey]
-					if(C.prefs.preferred_map)
-						if(choices[C.prefs.preferred_map]) //No votes if the map isn't in the vote.
-							var/preferred_map = C.prefs.preferred_map
-							choices[preferred_map] += 1
-							greatest_votes = max(greatest_votes, choices[preferred_map])
-					else if(config.defaultmap)
-						if(choices[config.defaultmap]) //No votes if the map isn't in the vote.
-							var/default_map = config.defaultmap.map_name
-							choices[default_map] += 1
-							greatest_votes = max(greatest_votes, choices[default_map])
-	//get all options with that many votes and return them in a list
+					var/preferred_map = C.prefs.preferred_map
+					if(!preferred_map)
+						preferred_map = global.config.defaultmap.map_name
```

Comment:
defaultmap can be null depending on config.

Reply:
Would you prefer it how it was previously? This was @LemonInTheDark's suggestion

Reply:
Just add continue if global.config.defaultmap is null

Reply:
done